tools/tf2openapi/types/tftensor.go

```diff
@@ -111,6 +111,53 @@ func NewTFDType(name string, dType string) (TFDType, error) {
 	return tfDType, nil
 }
 
-func (t *TFTensor) Schema() *openapi3.Schema {
-	return &openapi3.Schema{}
+func (t *TFTensor) RowSchema() *openapi3.Schema {
+	// ignore the 0th dimension because it is always -1 in row fmt to represent batchable input
+	return schema(1, t.Shape, t.Rank, t.DType)
+}
+
+func (t *TFTensor) ColSchema() *openapi3.Schema {
+	if t.Rank == -1 {
+		return openapi3.NewSchema()
+	}
+	return schema(0, t.Shape, t.Rank, t.DType)
+}
+
+func schema(dim int64, shape TFShape, rank int64, dType TFDType) *openapi3.Schema {
+	if dim == rank {
+		return dType.Schema()
+	}
+	if shape[dim] == -1 {
+		return openapi3.NewArraySchema().WithItems(schema(dim+1, shape, rank, dType))
+	}
+	if shape[dim] == 0 {
+		return openapi3.NewArraySchema().WithMaxItems(0)
+	}
+	return openapi3.NewArraySchema().WithMinItems(shape[dim]).WithMaxItems(shape[dim]).WithItems(schema(dim+1, shape, rank, dType))
+}
+
+func (t *TFDType) Schema() *openapi3.Schema {
```

Comment:
Maybe this would be resolved if we made our own file here (as you've already discussed)